examples/httpListen.js

```diff
@@ -0,0 +1,76 @@
+console.log = Sidevm.inspect;
```

Comment:
OK. I thought by default, chrome and node.js are both providing `inspect` as the log implementation, instead of the original one. Do you think performance is a concern for not using inspect as the default log impl?

Reply:
> Do you think performance is a concern for not using inspect as the default log impl?

Yes. It's one of the consideration. Another is to be compatible with the original QuickJS behavior.